docs/module.md

```diff
@@ -1,341 +1,246 @@
+not copy the contents of the module to the location of the `open`. This does not
+work for example:
 
 ```reason
-module BaseComponent = {
-  let defaultGreeting = "Hello";
-  let getAudience = (~excited) => excited ? "world!" : "world";
+module DurationCopy = {
+  open Duration;
+  let ten = fromSeconds(10);
 };
 
-module ActualComponent = {
-  /* the content is copied over */
-  include BaseComponent;
-  /* overrides BaseComponent.defaultGreeting */
-  let defaultGreeting = "Hey";
-  let render = () => defaultGreeting ++ " " ++ getAudience(~excited=true);
-};
+/* Error Unbound Value */
+let twenty = DurationCopy.fromSeconds(20);
 ```
 
-**Note**: `open` and `include` are very different! The former brings a module's content into your current scope, so that you don't have to refer to a value by prefixing it with the module's name every time. The latter **copies over** the definition of a module statically, then also does an `open`.
+A different keyword, `include`, does provide this copy behavior.
+It is covered [later](#including-modules).
 
-### Every `.re` file is a module
+## Including Modules
 
-An OCaml/Reason file maps to a module; this unlocks some interesting
-expressivity that'd previously require code generation in other languages. The
-file `React.re` implicitly forms a module `React`, which can be seen by other
-source files.
+The `include` keyword copies the contents of a module into another module. This
+can provide a sense of inheritance by always including a base module.
 
 ```reason
-/* FileA.re. This typically compiles to module FileA below */
-let a = 1;
-let b = 2;
-
-/* FileB.re */
-/* Maps FileA's implementation to a new API */
-let alpha = FileA.a;
-let beta = FileA.b;
-```
-
-This contrived snippet expresses "copying" a file:
+module BaseItem = {
+  let name = "Unknown";
+  let cost = 0;
+  let quantity = 0;
 
-```reason
-/* FileA.re. This typically compiles to module FileA below */
-let a = 1;
-let b = 2;
-
-/* FileB.re */
-/* compiles to exactly fileA.re's content with no runtime overhead! */
-include FileA;
-```
+  let getTotalCost = () => cost * quantity;
+};
 
-**Note**: Because files are modules, file names should, by convention, be capitalized so they match their module names. Uncapitalized file names are not invalid, but will be transformed into a capitalized module name. I.e. `file.re` will be compiled into the module `File`. To simplify and minimize the disconnect here, the convention is therefore to capitalize file names too.
+module Apples = {
+  include BaseItem;
 
-Signatures
-----------------------------------
+  /* These overwrite the values from BaseItem */
+  let name = "Apple";
+  let cost = 50;
+  let quantity = 7;
+};
 
-A module's type is called a "signature", and can be written explicitly. If a
-module is like a `.re` (implementation) file, then a module's signature is like
-a `.rei` (interface) file.
+/* We do not have to re-implement this function for each item. */
+Apples.getTotalCost();
```

Comment:
It was nice that you showed how new bindings overwrite the previous ones, which is sometimes what you want, however, this comment here could be misleading "We do not have to re-implement this function for each item. " because in this case, the old `getTotalCost()` references the old values not the overridden ones. There's a good chance the user _does_ have to re-implement this function _in this case_. Maybe consider rewording to:
> `Apples` inherits `getTotalCost`, but `getTotalCost` still refers to the _previous_ versions of `name`, `cost`, and `quantity`.

Reply:
Nice catch, I did it backwards! I'll update this example to something that makes more sense.